include/Utils/DIPUtils.h

```diff
@@ -491,6 +511,253 @@ void BilinearInterpolationResizing(
       });
 }
 
+// Util function for erosion doesn't support tail processing
+void compAndStorewoTailProcessingerosion(OpBuilder &builder, Location loc,
```

Comment:
Consider renaming it to `calcAndStorewoTailProcessingErosion` for uniformity in naming convention. Similar advice for `compAndStorewTailProcessingErosion` and other related functions as well.